src/workerd/jsg/BUILD.bazel

```diff
@@ -121,6 +120,7 @@ wd_cc_library(
         ":meta",
         ":modules_capnp",
         ":observer",
+        ":url",
```

Comment:
This is the old url implementation. Any reason why we are not using the new one?

Reply:
No, this is the JSG url package.